doc/guides/debug_tools/coredump.rst

```diff
@@ -28,6 +28,10 @@ Here are the choices regarding memory dump:
   walking the stack in the debugger. Use this only if absolute minimum of data
   dump is desired.
 
+* ``DEBUG_COREDUMP_MEMORY_DUMP_SELECT``: dumps the bare minimum memory content
+  that would be included with ``DEBUG_COREDUMP_MEMORY_DUMP_MIN`` plus any
+  additional memory regions that were registered for collection.
```

Comment:
I think the documentation lacks description of how to register additional memory regions to be dumped.

How are users expected to know how to use this feature ?

I believe some references to `coredump_register_memory_region()` / API is a bare minimum here.